var/spack/repos/builtin/packages/py-matplotlibscalebar/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyMatplotlibscalebar(PythonPackage):
+    """Provides a new artist for matplotlib to display a scale bar,
+    aka micron bar."""
+
+    homepage = "https://github.com/ppinard/matplotlib-scalebar"
+    pypi = "matplotlib-scalebar/matplotlib-scalebar-0.6.1.tar.gz"
+    git = "https://github.com/ppinard/matplotlib-scalebar"
+
+    version("develop", git=git, branch="master")
```

Comment:
```suggestion
    version("develop", branch="master")
```

Reply:
We should also call the version master instead of changing the name